debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -386,6 +386,17 @@ public static EventDeserializationFailureHandlingMode parse(String value) {
      */
     private static final int DEFAULT_BINLOG_BUFFER_SIZE = 0;
 
+    /**
+     * Default length of interval in which BinlogReader generates periodically
+     * heartbeat messages. A size of 0 disables heartbeat.
+     */
+    private static final int DEFAULT_BINLOG_READER_HEARTBEAT_INTERVAL = 3600;
```

Comment:
I'd feel better if we disabled it by default, the reason being that it'd break the setup of someone without auto topic creation. I.e. after the update we'd suddenly try to write to a topic which doesn't exist. I think this should be opt-in.